### PR TITLE
Fix view op slicing for 2nd dim in case of 0 offset

### PR DESCRIPTION
**Pull request by DenisVieriu97**
Fixes slicing of view ops when both dimensions need to be sliced and the 2nd dimension to be sliced has an offset of 0.
e.g in case of base shape `[3, 4]` and view shape `[1, 2] storrage_offset=8`, both dimensions of the base shape would need to be sliced. The offset for the first element in the 2nd dimension is 0, which was causing the previous check ` if (nextSliceOffset)` to fail. This change fixes the condition so an offset of 0 won't be ignored.

### CODE REVIEW

File: test/test_mps.py [hunk 1]
```diff
@@ -1841,6 +1841,17 @@ def test_slice_reshape_contg_view(self):
 
         self.assertEqual(r_mps, r_cpu)
 
+    def test_contiguous_slice(self):
+        t_mps = torch.randn([3, 4], device="mps")
+        t_cpu = t_mps.detach().clone().cpu()
+
```
Comment (kulinseth) [inline test/test_mps.py:1847]: Can you add the test case where the storage offset is slicing 3D , 4D tensor with slice offset ?

**Review by kulinseth — APPROVED** [review]
